packages/main/src/components/ObjectStatus/ObjectStatus.jss.ts

```diff
@@ -54,7 +54,7 @@ const styles = {
     height: '1rem',
     lineHeight: '1rem',
     verticalAlign: 'top',
-    paddingRight: '0.25rem',
+    paddingInlineEnd: '0.25rem',
 
     '&:only-child': {
       paddingRight: 0
```

Comment:
```suggestion
      paddingInlineEnd: 0
```
This should be `paddingInlineEnd` as well, otherwise the overwrite will not work for RTL